src/explore-education-statistics-common/src/modules/table-tool/components/Wizard.tsx

```diff
@@ -59,7 +61,7 @@ const Wizard = ({
   const mountedRef = useMountedRef();
 
   const [shouldScroll, setShouldScroll] = useState(scrollOnMount);
-  const [currentStep, setCurrentStepState] = useState(initialStep);
+  const [activeStep, setActiveStepState] = useState(initialStep);
```

Comment:
Minor, but if we're renaming this to `activeStep`, then the setter could be changed to `setActiveStep` too

Reply:
ok, renamed now.